pkg/helmcertifier/checks/checks_test.go

```diff
@@ -30,34 +31,28 @@ func TestIsHelmV3(t *testing.T) {
 	}
 
 	positiveTestCases := []testCase{
-		{description: "valid tarball, absolute path", uri: "/tmp/chart-v3.tgz"},
-		{description: "valid tarball, relative path", uri: "./chart-v3.tgz"},
-		{description: "valid tarball, http", uri: "http://www.example.com/chart-v3.tgz"},
-		{description: "valid directory, absolute path", uri: "/tmp/chart-v3"},
-		{description: "valid directory, relative path", uri: "./chart-v3"},
+		{description: "valid tarball, absolute path", uri: "testchart-0.1.0.tgz"},
 	}
 
 	for _, tc := range positiveTestCases {
 		t.Run(tc.description, func(t *testing.T) {
 			r, err := IsHelmV3(tc.uri)
 			require.NoError(t, err)
 			require.NotNil(t, r)
+			require.Equal(t, true, r.Ok)
 		})
 	}
 
 	negativeTestCases := []testCase{
-		{description: "invalid tarball, absolute path", uri: "/tmp/chart-v2.tgz"},
-		{description: "invalid tarball, relative path", uri: "./chart-v2.tgz"},
-		{description: "invalid tarball, http", uri: "http://www.example.com/chart-v2.tgz"},
-		{description: "invalid directory, absolute path", uri: "/tmp/chart-v2"},
-		{description: "invalid directory, relative path", uri: "./chart-v2"},
+		{description: "invalid tarball, absolute path", uri: "helm2testchart-0.1.0.tgz"},
```

Comment:
The intention of those tests were to be used to guide the development, being either part of this changelist or be picked up by someone else, so please revert the change and amend the proper test case.

Reply:
Verifying various kinds of URI patterns need not be part of every check. A separate test would be sufficient for the same.